src/libraries/Microsoft.PowerFx.Connectors/OpenApiExtensions.cs

```diff
@@ -691,7 +691,7 @@ internal static ConnectorType GetConnectorType(this ISwaggerParameter openApiPar
                     return new ConnectorType(schema, openApiParameter, FormulaType.Blob);
 
                 default:
-                    return new ConnectorType(error: $"Unsupported schema type {schema.Type}");
+                    return new ConnectorType(error: $"Unsupported schema type {schema.Type} for field {openApiParameter.Name}");
             }
```

Comment:
should we set name / displayname here?

Reply:
but you're not failing ... you're passing a partially initialized object through the rest of the system and letting it fail at some future point in totally unpredictable ways.  

So the current test shows us this will hit a null ref later ... and you added an extra null check in ConnectorType to protect against that one failure ...

Reply:
Not here, but later with a null ref